README.md

```diff
@@ -0,0 +1,50 @@
+# Jakarta EE 10 Examples
+
+
+This repository contains code examples for Jakarta EE 10.
+
+A selection of these examples have an elaborate explanation in the Jakarta EE Tutorial.
+
+The Jakarta EE Tutorial code examples is located in a separate repository
```

Comment:
Is this accurate? Aren't they in this repo now?

Reply:
It was a typo, sorry for that